src/client.ts

```diff
@@ -88,16 +125,18 @@ class Client {
   connect(deviceId, cb) {
     // User may "re-connect" if a device ID has previously been stored
     if (typeof deviceId === 'function') {
-      if (!this.deviceId) 
-        return cb('No device ID has been stored. Please connect with your device ID first.')
+      if (!this.deviceId)
+        return cb(
+          'No device ID has been stored. Please connect with your device ID first.'
+        );
       cb = deviceId;
     } else {
       // If the user passes in a device ID, connect to that device and save
       // the new ID for future use.
       this.deviceId = deviceId;
     }
     const param = this._buildRequest(deviceCodes.CONNECT, this.pubKeyBytes());
-    this._request(param, null, (err, res) => {
+    this._request(param, (err, res) => {
```

Comment:
We need the `null`: https://github.com/GridPlus/gridplus-sdk/pull/240/files#diff-627fe08d8bba6a4fa76e7c5ed36ef6f16d126733e65e19236b391de0d34f1fe0R100